.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
- [ ] You have to list the `test.md` inside the `.gitignore` file. This is one of the requirements of this project that must be satisfied.